pact/Pact/Core/Literal.hs

```diff
@@ -19,13 +24,15 @@ import GHC.Generics
 
 import Pact.Core.Pretty
 
+deriving instance Data Decimal
```

Comment:
This orphan will go away if/when a PR I opened earlier to `data-decimal` will be merged.

I'm also happy to move this orphan to a separate orphans module, since we already have another orphan elsewhere, so we might as well isolate that.

Reply:
A `Todo` would be great, so that it does not get lost.